packages/tools/src/tools/ZoomTool.ts

```diff
@@ -38,6 +40,7 @@ class ZoomTool extends BaseTool {
     super(toolProps, defaultToolProps);
     this.initialMousePosWorld = [0, 0, 0];
     this.dirVec = [0, 0, 0];
+    this.initialParallelScales = new Map();
```

Comment:
Could you explain why we're keeping `initialParallelScales`? I'm not really following.

Reply:
if you look at the video in #1963, the initial parallelScale is lost if you set minZoomScale to 1, when you zoom in the first time it will set a new parallelScale, if you zoom out and call getZoom() you will get 1.242 instead of 1, so before doing anything we need to keep the initialScale.